core/src/main/java/com/novoda/noplayer/NoPlayer.java

```diff
@@ -135,6 +136,8 @@
 
     void disableAdverts();
 
+    void skipAdverts(@NonNull AdvertBreak advertBreak);
```

Comment:
I'd probably drop the `@NonNull` from all of these, I find this to be a redundant annotation. The implication should always be to pass something that is `@NonNull`

Reply:
this is not related to this PR, but we should consider adding `@NonNull` annotation to the public API because otherwise Kotlin thinks that all the objects there are nullable when it's not the case

Reply:
Makes sense 👍

Reply:
I'll keep the `@NonNull` on public ones